ios/Exponent/Versioned/Modules/Api/EXNotifications.m

```diff
@@ -40,6 +41,18 @@ - (void)setBridge:(RCTBridge *)bridge
   _experienceId = _bridge.experienceScope.experienceId;
 }
 
+
+RCT_REMAP_METHOD(getDevicePushTokenAsync,
+                 getDevicePushTokenAsyncWithResolver:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject)
+{
+  NSData *token = [[NSUserDefaults standardUserDefaults] apnsToken];
```

Comment:
Can we reject if `[EXShellManager sharedInstance].isShell` is false? This ensures that the method only works in Standalone (or ExpoKit) projects. It doesn't make sense in Expo Client since that app needs to sandbox many projects.